### DISCUSSION TITLE
Origin Not Allowed

### DISCUSSION THREAD

**Discussion by santhosh-stitch** [zitadel/zitadel#6363] [Q&A]
Hi, 

please help us on the following issue we are using zitadel custom ui and calling session creation api and the details of the api call as follows,

api - v2alpha/sessions 
method - post 
payload : { 
"checks": {     
         "user": {      
             "loginName": "our user email address"   
                      }  
                  } 
               } 
response : {     
            "code": 7,     
            "message": "Errors.OriginNotAllowed (AUTH-DZG21)",     
            "details": [         
                        {            
                           "@type": "type.googleapis.com/zitadel.v1.ErrorDetail",            
                           "id": "AUTH-DZG21",            
                           "message": "Errors.OriginNotAllowed"       
                       }   
                   ] } 

we could not find the actual solution for this error we have added the localhost:3000 our development url in the zitadel allowed origin url under the project but the error still exits

<img width="844" alt="image" src="https://github.com/zitadel/zitadel/assets/104059308/c4bae59b-e4a4-48c3-bab1-e1756be16e7e">

**Comment by fforootd** [comment]
I guess you already asked this here https://discord.com/channels/927474939156643850/927866013545025566/1140549541880401931

Please do not trigger multiple channels it only confuses people 😁
